tests/unittests/test_util.py

```diff
@@ -2200,7 +2200,7 @@ def test_parse_mount_with_ext(self, mount_out):
         self.assertEqual(("/dev/mapper/vg00-lv_root", "ext4", "/"), ret)
         # this one exists in mount_parse_ext.txt
         ret = util.parse_mount("/sys/kernel/debug")
-        self.assertIsNone(ret)
+        self.assertEqual(("none", "debugfs", "/sys/kernel/debug"), ret)
```

Comment:
why does this one change?

Reply:
before it didn't parse, i don't know why.
now it just does.